examples/bert/README.md

```diff
@@ -5,7 +5,6 @@ This folder contains examples of BERT optimization using different workflows.
 - CPU: [Optimization with Intel® Neural Compressor PTQ](#bert-optimization-with-intel®-neural-compressor-ptq-on-cpu)
 - CPU: [Optimization with QAT Customized Training Loop](#bert-optimization-with-qat-customized-training-loop-on-cpu)
 - GPU: [Optimization with CUDA/TensorRT](#bert-optimization-with-cudatensorrt-on-gpu)
-- GPU: [Optimization with TensorRT-Model-Optimizer](#bert-optimization-with-tensorRT-model-optimizer-on-cpugpu)

```

Comment:
Please remove this example from the doc here too https://github.com/microsoft/Olive/blame/5925e72dfe42ef2f8a2f0f4a46eed3b7182e2fb2/docs/source/examples.md#L11

Reply:
Oh my bad, i assumed the trt ep referred to the nvmo example but looks like it's referring to the cuda/tensorrt example above.